src/exchange/sockets.ts

```diff
@@ -209,4 +223,31 @@ export class Sockets {
   ) => {
     this.socketInstance.on(SOCKET_EVENTS.AccountDataUpdateKey, cb);
   };
+
+
+  async onDisconnect(): Promise<void> {
```

Comment:
Should we add similar methods in websockets?

Reply:
Honestly, not even sure if it is being used by anyone anywhere 😆  so reconnection is something I would first have to research, setup and test before making changes. Did that for socket.io but not sure if doing same research for UWebsocket would be of highest priority as of now.